backend/src/main/App.ts

```diff
@@ -23,62 +27,11 @@ createConnection()
     app.use(bodyParser.json());
     const PORT = 8000;
 
-    const userRepository: UserRepository = getCustomRepository(UserRepository);
-    const promotionRepository: PromotionRepository = getCustomRepository(
-      PromotionRepository
-    );
-    const discountRepository: DiscountRepository = getCustomRepository(
-      DiscountRepository
-    );
-    const savedPromotionRepository: SavedPromotionRepository = getCustomRepository(
-      SavedPromotionRepository
-    );
+    // await loadSampleData();
```

Comment:
Commented this out for now, for me when developing locally it was easier to load the data only **once.** So synchornize is true, dropSchema is false. That way we can leverage nodemon, otherwise nodemon would keep executing this line everytime we save.